pixeltable/io/label_studio.py

```diff
@@ -152,20 +157,22 @@ def __update_tasks(self, t: Table, existing_tasks: dict[tuple, dict]) -> SyncSta
         config = self.__project_config
 
         # Columns in `t` that map to Label Studio data keys
-        t_data_cols = [t_col for t_col, ext_col_name in self.col_mapping.items() if ext_col_name in config.data_keys]
+        t_data_cols = [
```

Comment:
Why not pass ColHandles around everywhere instead? That would make it easier to follow.